input_output/file_output.py

```diff
@@ -0,0 +1,33 @@
+from supporting_components.graph_io import write_dot
+import subprocess
+
+
+"""
+Before you can use this module you must install graphviz on the terminal to be able to do this:
+    https://stackoverflow.com/questions/1494492/graphviz-how-to-go-from-dot-to-a-graph
+Install instructions:
+    https://www.graphviz.org/download/
+"""
+
+
+def save_graph_as_dot(G, filename):
+    """
+    This method saves a graph in a dot file  .
+    :param G: The graph to be saved.
+    :param filename: The filename in which the graph should be saved.
+    """
+
+    dot_filename = '../output_graphs/dot/' + filename + '.dot'
+    with open(dot_filename, 'w') as g0:
+        write_dot(G, g0)
+
+
+def save_graph_in_png(filename):
+    """
+    This method can use a .dot file and transform it into a png image of the graph.
+    :param filename: The filename of the .dot file without the extension, this will also be the filename for the .png file
+    """
+
+    dot_filename = '../output_graphs/dot/' + filename + '.dot'
```

Comment:
These path names are relative. I'm not sure if we're allowed to do that

[It only runs in my PyCharm when I remove all ../]

(from the reader canvas):

> Ensure that your program runs by putting all modules, together with the test in-
> stances in the same directory (ensure that the Python path setting does not matter,
> and that you do not include path names when reading les in your program).

Reply:
Ja, je hebt gelijk. Heb nu in de file_output toegevoegd dat hij eerst de ROOT moet bepalen (omdat de file_output altijd '../' van de root vandaan is moet dat goed gaan)

Reply:
Maar we hebben nu niet dat de test_graphs in dezelfde map staan. Weet niet of dat een probleem gaat worden later

Reply:
aanpassing werkt!